pkgs/applications/graphics/qimgv/default.nix

```diff
@@ -1,6 +1,15 @@
-{ mkDerivation, fetchFromGitHub, lib
-, pkgconfig, cmake
-, exiv2, qtbase, qtimageformats, qtsvg
+{ mkDerivation
+, lib
+, fetchFromGitHub
+
+, cmake
+, pkgconfig
+
+, exiv2
+, mpv
+, qtbase
+, qtimageformats
+, qtsvg
```

Comment:
I'm firmly in the "don't really care either way" camp, but to collapse it back down means I have to disable `format-all-mode` before I save in Emacs, which is a pain since I like the other cleanup `nixpkgs-format` does for me. And, since I'm the sole maintainer, I don't imagine many others will be judging the organization (until the next PR, that is).

Thanks for the feedback, though :)